.eslintrc.js

```diff
@@ -47,6 +50,17 @@ module.exports = {
             "always", {
                 "avoidQuotes": true
             }
+        ],
+        "brace-style": [
+            "error",            
+            "stroustrup",
+            {"allowSingleLine": true}
```

Comment:
can you open this up onto a new line? like the "always" and "avoidQuotes" above?

```
"stroustrup", {
    "allowSingleLine": true}
}
```